src/components/ProjectSettingsTabs/GithubCommitQueueTab/getFormSchema.ts

```diff
@@ -3,7 +3,12 @@ import { SpruceFormProps } from "components/SpruceForm";
 import { CardFieldTemplate } from "components/SpruceForm/FieldTemplates";
```

Comment:
I noticed that the "Save changes on Page" button is enabled even after I refresh the page. I feel like it should be disabled at that point. Am I doing something wrong or is there a bug?

Reply:
It is a bug :( good catch!

Reply:
I did some digging and I believe this bug is a result of a bug in the form library: https://github.com/rjsf-team/react-jsonschema-form/issues/1201. The onChange handler fires many times on mount, causing the "hasChanges" flag to be set to `true` and the save button to be enabled.

I've come up with a better way to manage saveability but I think it's worth doing in another ticket: EVG-16268. Will merge this as-is!